helm-chart/renku-notebooks/templates/registry-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- $auth := printf "%s:%s" .Values.gitlab.registry.username .Values.gitlab.registry.token | b64enc -}}
+{{- $registry := .Values.gitlab.registry.host | replace "http://" "" | replace "https://" "" | replace ":443" "" -}}
+{{- $secret := printf "{\"auths\":{\"%s\":{\"username\":\"%s\",\"password\":\"%s\",\"email\":\"root@example.com\",\"auth\":\"%s\"}}}" $registry .Values.gitlab.registry.username .Values.gitlab.registry.token $auth -}}
```

Comment:
Does that email have to be here? It could be any admin, no?

Reply:
true I hadn't noticed that. It doesn't actually seem to match the email in gitlab for the root user so it doesn't look like it's being used. I would leave it for now.